pkg/apis/serving/v1alpha1/servingruntime_types.go

```diff
@@ -21,7 +21,8 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-type Framework struct {
+// +k8s:openapi-gen=true
+type ServingRuntimeFramework struct {
```

Comment:
Per slack discussion we'd like to rename this to `SupportedModelFormat` as currently we use a mixed list of model frameworks like `tensorflow`, `pytorch`, `sklearn`, `xgboost` and model formats `onnx`, `pmml`, `torchscript`, `tensorrt`, so `ModelFormat` is more inclusive.